modules/Microsoft.Storage/storageAccounts/deploy.bicep

```diff
@@ -201,6 +201,12 @@ param cMKKeyVersion string = ''
 @description('Optional. The name of the diagnostic setting, if deployed. If left empty, it defaults to "<resourceName>-diagnosticSettings".')
 param diagnosticSettingsName string = ''
 
+@description('Optional. The SAS expiration action. Can only be Log.')
+param expirationAction string = 'Log'
```

Comment:
If this can only be log, I think we should ever hardcode it below, or add an allowedValues block. The first may be there preferred choice